core-bundle/src/Resources/public/core.js

```diff
@@ -431,6 +431,127 @@ var AjaxRequest =
 		}).post({'action':'toggleSubpalette', 'id':id, 'field':field, 'load':1, 'state':1, 'REQUEST_TOKEN':Contao.request_token});
 	},
 
+	/**
+	 * Toggle state of a checkbox field
+	 *
+	 * @param {object}  el            The DOM element
+	 * @param {boolean} rowIcon Whether the row icon should be toggled as well
+	 *
+	 * @returns {boolean}
+	 */
+	toggleField: function(el, rowIcon) {
+		el.blur();
+
+		var img = null,
+			image = $(el).getFirst('img'),
+			published = (image.get('data-state') == 1),
+			div = el.getParent('div'),
+			index, next, icon, icond, pa, params;
```

Comment:
```suggestion
			next, icon, icond, pa;
```

It seems you are not using `index` and `params`?

Reply:
Fixed in 818a5f18c87cf97383fdac47406feecb1626e553